src/lib/fcitx/action.h

```diff
@@ -139,6 +140,14 @@ class FCITXCORE_EXPORT Action : public Element {
 
     FCITX_DECLARE_SIGNAL(Action, Update, void(InputContext *));
 
+    /**
```

Comment:
No, you can't add new virtual function without break ABI.

Let's just say we assume this is getter/setter type value and store it within ActionPrivate.

Reply:
Also, in the documentation, mention this is for "Display only" if UI implementation supports it, and doesn't mean anything within the key handling logic.